documentation/docs/01-routing.md

```diff
@@ -43,7 +43,9 @@ Dynamic parameters are encoded using `[brackets]`. For example, a blog post migh
 
 ### Endpoints
 
-Endpoints are modules written in `.js` (or `.ts`) files that export functions corresponding to HTTP methods. For example, our hypothetical blog page, `/blog/cool-article`, might request data from `/blog/cool-article.json`, which could be represented by a `src/routes/blog/[slug].json.js` endpoint:
+Endpoints are modules written in `.js` (or `.ts`) files that export functions corresponding to HTTP methods. All endpoints have access to `fetch` globally without needing to import an external dependency.
```

Comment:
It's not just endpoints with access to `fetch`, but the whole application, right? Hooks are the other place I could possibly see using it (e.g. to send metrics to an external server)